### PR TITLE
Add package libwpg version 0.3.3

### PR DESCRIPTION
**Pull request by suresh-thelkar**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Add package libwpg version 0.3.3

###### Change Log  

- NA

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

Buddy Build
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293986&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=293987&view=results)

### CODE REVIEW

File: SPECS-EXTENDED/libwpg/libwpg.spec [hunk 1]
```diff
@@ -0,0 +1,229 @@
+Summary:        A library for import of WordPerfect Graphics images
+Name:           libwpg
+Version:        0.3.3
+Release:        9%{?dist}
+License:        LGPL-2.0-or-later OR MPL-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://libwpg.sourceforge.net/
+Source0:        http://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```
Comment (PawelWMS) [inline SPECS-EXTENDED/libwpg/libwpg.spec:9]: Let's use HTTPS whenever possible:
```suggestion
Source0:        https://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```

File: SPECS-EXTENDED/libwpg/libwpg.spec [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+%global         apiversion 0.3
```
Comment (PawelWMS) [inline SPECS-EXTENDED/libwpg/libwpg.spec:10]: We're trying to keep the macros/globals/defines at the top to make them easier to find. And yes, the linter is still stubbornly moving them around.:)

```suggestion
%global         apiversion 0.3

Summary:        A library for import of WordPerfect Graphics images
Name:           libwpg
Version:        0.3.3
Release:        9%{?dist}
License:        LGPL-2.0-or-later OR MPL-2.0
Vendor:         Microsoft Corporation
Distribution:   Mariner
URL:            https://libwpg.sourceforge.net/
Source0:        http://download.sourceforge.net/libwpg/%{name}-%{version}.tar.xz
```

File: SPECS-EXTENDED/libwpg/libwpg.spec [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+%build
+%configure --disable-static --disable-silent-rules
+sed -i \
+    -e 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' \
+    -e 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' \
+    libtool
+make %{?_smp_mflags}
+
+%install
+make install DESTDIR=%{buildroot}
```
Comment (PawelWMS) [inline SPECS-EXTENDED/libwpg/libwpg.spec:69]:
```suggestion
%make_build

%install
%make_install
```

**Review by PawelWMS — APPROVED** [review]
A few small requests but otherwise looks good.

**Comment by suresh-thelkar** [comment]
> A few small requests but otherwise looks good.

Thank for your review and approval. I have implemented your suggestions and ran Buddy builds after that. Here are the results. 
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=294649&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=294650&view=results)

**Pull request merged**